packages/gestalt/src/SheetMobile.js

```diff
@@ -0,0 +1,183 @@
+// @flow strict
+import { type ElementConfig, type Node } from 'react';
+import Button from './Button.js';
+import Link from './Link.js';
+import AnimationProvider from './animation/AnimationContext.js';
+import DismissingElement from './animation/DismissingElement.js';
+import { useDeviceType } from './contexts/DeviceTypeProvider.js';
+import FullPage from './SheetMobile/FullPage.js';
+import PartialPage from './SheetMobile/PartialPage.js';
+
+type OnClickType = ({|
+  event:
+    | SyntheticMouseEvent<HTMLButtonElement>
+    | SyntheticKeyboardEvent<HTMLButtonElement>
+    | SyntheticMouseEvent<HTMLAnchorElement>
+    | SyntheticKeyboardEvent<HTMLAnchorElement>,
+  onDismissStart: () => void,
+|}) => void;
+
+type Props = {|
+  /**
+   * String that clients such as VoiceOver will read to describe SheetMobile when opened. See [Accessibility section](https://gestalt.pinterest.systems/web/sheetmobile#Accessibility) for more info.
+   */
+  accessibilityLabel?: string,
+  /**
+   * Specify the alignment of `heading` & `subHeading` strings. See the [Header variant](https://gestalt.pinterest.systems/web/sheetmobile#Heading) for more info.
+   */
+  align?: 'start' | 'center',
```

Comment:
I think this should prop should be header alignment. Or maybe, this entire thing can be called

```
header = {title:'string', subtitle?:'string', align?:'center'}
```

Reply:
DONE

Reply:
Hovewer, setting defaults in objects don't get captured in the prop tables.

Reply:
Also, modal follows this pattern too. I'd reconsider amending all at once.

Reply:
Personally, I also like the idea of adding everything else like buttons and IconButtons to the props as well.

Just don't know if that would make our prop table a tad harder to read. Agreed, if we do this here, then doing in the Modal makes sense. But might as well introduce it here, and maybe we can reuse the header across Modal and Sheet